packages/plugins/documentation/preview-output/showcase/models/Task.md

```
@@ -0,0 +1,258 @@
+        String title
+        String body
+        String slug
+        TaskStatus status
+        Priority priority
+        DateTime dueDate
+        Float estimatedHours
+        Int commentCount
+        String projectId FK
+        String assigneeId FK
+        String parentId FK
+    }
```

markdownlint-cli2 (0.21.0)

[warning] Blank line inside blockquote

(MD028, no-blanks-blockquote)